.yarn/patches/@metamask-assets-controllers-npm-43.1.1-c223d56176.patch

```diff
@@ -0,0 +1,14 @@
+diff --git a/dist/token-prices-service/codefi-v2.mjs b/dist/token-prices-service/codefi-v2.mjs
```

Comment:
Can we remove this patch and rebase off of `develop`?

Reply:
To be honest, I'm not sure. I will need to sync with Brian and Salim tomorrow, but hopefully we can just bump to the actual version 44 instead of the patch.

Reply:
Let's talk about this during standup. Isolating the token detection changes probably makes the most sense here.